firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -135,3 +137,57 @@ def test_matching_text(self):
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
                 self.assertIn(match_fragment, (input_text, input_text.upper()))
+
+
+class TestIdentityPopup(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
```

Comment:
Don't forget to add the tearDown, which force closes the popup. I assume you will add it once the close() method has been landed via the other PR.